app/i18n/nl/gen.php

```diff
@@ -0,0 +1,166 @@
+		'login_persona' => 'Login met Persona',
+		'login_persona_problem' => 'Connectiviteits problemen met Persona?',
+		'logout' => 'Log uit',
+		'password' => 'Wachtwoord',
+		'reset' => 'Authenticatie reset',
+		'username' => 'Gebruikersnaam',
+		'username_admin' => 'Administrator gebruikersnaam',
+		'will_reset' => 'Het authenticatie system zal worden gereset: een formulier zal worden gebruikt in plaats van Persona.',
+	),
+	'date' => array(
+		'Apr' => '\\A\\p\\r\\i\\l',
+		'Aug' => '\\A\\u\\g\\u\\s\\t\\u\\s',
+		'Dec' => '\\D\\e\\c\\e\\m\\b\\e\\r',
+		'Feb' => '\\F\\e\\b\\r\\u\\a\\r\\i',
+		'Jan' => '\\J\\a\\n\\u\\a\\r\\i',
+		'Jul' => '\\J\\u\\l\\i',
+		'Jun' => '\\J\\u\\n\\i',
+		'Mar' => '\\M\\a\\a\\r\\t',
+		'May' => '\\M\\e\\i',
+		'Nov' => '\\N\\o\\v\\e\\m\\b\\e\\r',
+		'Oct' => '\\O\\k\\t\\o\\b\\e\\r',
+		'Sep' => '\\S\\e\\p\\t\\e\\m\\b\\e\\r',
+		'apr' => 'apr',
+		'april' => 'Apr',
+		'aug' => 'aug',
+		'august' => 'Aug',
+		'before_yesterday' => 'Ouder',
+		'dec' => 'dec',
+		'december' => 'Dec',
+		'feb' => 'feb',
+		'february' => 'Feb',
+//		'format_date' => '%s j\\<\\s\\u\\p\\>S\\<\\/\\s\\u\\p\\> Y',
+		'format_date' => 'j %s Y', // European date format
+//		'format_date_hour' => '%s j\\<\\s\\u\\p\\>S\\<\\/\\s\\u\\p\\> Y \\a\\t H\\:i',
+		'format_date_hour' => 'j %s Y \\o\\m H\\:i', // European date format
+		'fri' => 'Vr',
+		'jan' => 'jan',
+		'january' => 'Jan',
+		'jul' => 'jul',
+		'july' => 'Jul',
+		'jun' => 'jun',
+		'june' => 'Jun',
+		'last_3_month' => 'Laatste drie maanden',
+		'last_6_month' => 'Laatste zes maanden',
+		'last_month' => 'Vorige maand',
+		'last_week' => 'Vorige week',
+		'last_year' => 'Vorig jaar',
+		'mar' => 'mar',
+		'march' => 'Mar',
+		'may' => 'Mei',
+		'mon' => 'Ma',
+		'month' => 'maanden',
+		'nov' => 'nov',
+		'november' => 'Nov',
+		'oct' => 'okt',
+		'october' => 'Okt',
+		'sat' => 'Za',
+		'sep' => 'sep',
+		'september' => 'Sep',
+		'sun' => 'Zo',
+		'thu' => 'Do',
+		'today' => 'Vandaag',
+		'tue' => 'Di',
+		'wed' => 'Wo',
+		'yesterday' => 'Gisteren',
+	),
+	'freshrss' => array(
+		'_' => 'FreshRSS',
+		'about' => 'Over FreshRSS',
+	),
+	'js' => array(
+		'category_empty' => 'Lege categorie',
+		'confirm_action' => 'Weet u zeker dat u dit wilt doen? Het kan niet ongedaan worden gemaakt!',
+		'confirm_action_feed_cat' => 'Weet u zeker dat u dit wilt doen? U verliest alle gereleteerde favorieten en gebruikers informatie. Het kan niet ongedaan worden gemaakt!',
+		'feedback' => array(
+			'body_new_articles' => 'Er zijn \\d nieuwe artikelen om te lezen op FreshRSS.',
+			'request_failed' => 'Een opdracht is mislukt, mogelijk door Internet verbindings problemen.',
+			'title_new_articles' => 'FreshRSS: nieuwe artikelen!',
+		),
+		'new_article' => 'Er zijn nieuwe artikelen beschikbaar, klik om de pagina te vernieuwen.',
+		'should_be_activated' => 'JavaScript moet aan staan',
+	),
+	'lang' => array(
+		'de' => 'Deutsch',
+		'en' => 'English',
+		'fr' => 'Français',
+		'nl' => 'Nederlands',
```

Comment:
Done https://github.com/Alkarex/FreshRSS/commit/501e3312101893b147767c9f3a36c947869d6580
Maybe in the future we could extract this data, which is the same for all languages.